packages/oauth/oauth-provider-ui/src/components/utils/scope-description.tsx

```diff
@@ -0,0 +1,529 @@
+      title={t`Storage`}
+      description={t`Upload files`}
+    />
+  )
+}
+
+function AccountPermissions({
+  permissions,
+}: {
+  permissions: PermissionSetTransition
+}) {
+  const { t } = useLingui()
+
+  if (permissions.allowsAccount({ attr: 'status', action: 'manage' })) {
+    return (
+      <DescriptionCard
+        role="listitem"
+        image={<AccountOutlinedIcon className="size-6" />}
+        title={t`Account`}
+        description={t`Manage your account status`}
+      />
+    )
+  }
+
+  return null
+}
+
+const isChatNsid = (nsid: NSID | '*'): nsid is '*' | `chat.bsky.${string}` =>
+  nsid === '*' || nsid.startsWith('chat.bsky.')
+const isChatRpcScope = (s: string): boolean =>
+  RpcScope.fromString(s)?.lxm.some(isChatNsid) || false
+
+function ChatPermissions({
+  permissions,
+}: {
+  permissions: PermissionSetTransition
+}) {
+  const { t } = useLingui()
+
+  const canChat = useMemo(() => {
+    return (
+      permissions.hasTransitionChatBsky ||
+      permissions.scopes.some(isChatRpcScope)
+    )
+  }, [permissions])
+
+  if (canChat) {
+    return (
+      <DescriptionCard
+        role="listitem"
+        image={<ChatIcon className="size-6" />}
+        title={t`Chat`}
+        description={t`Read and send messages`}
+      />
+    )
+  }
+
+  return null
+}
+
+function IdentityPermissions({
+  permissions,
+}: {
+  permissions: PermissionSetTransition
+}) {
+  const { t } = useLingui()
+
+  if (
+    permissions.allowsIdentity({ attr: '*', action: 'manage' }) ||
+    permissions.allowsIdentity({ attr: '*', action: 'submit' })
+  ) {
+    return (
+      <DescriptionCard
+        role="listitem"
+        image={<AtSymbolIcon className="h-6" />}
+        title={t`Identity`}
+        description={t`Manage your full identity (including your @handle)`}
+      />
+    )
+  }
+
+  if (permissions.allowsIdentity({ attr: 'handle', action: 'manage' })) {
+    return (
+      <DescriptionCard
+        role="listitem"
+        image={<AtSymbolIcon className="size-6" />}
+        title={t`Handle`}
+        description={t`Update your network @handle`}
+      />
+    )
+  }
+
+  return null
+}
+
+function RpcMethodsDetails({
+  permissions,
+}: {
+  permissions: PermissionSetTransition
+}) {
+  const { t } = useLingui()
+
+  if (permissions.hasTransitionGeneric) {
+    return (
+      <DescriptionCard
+        role="listitem"
+        image={<AuthenticateIcon className="size-6" />}
+        title={t`Authenticate`}
+        description={t`Perform authenticated actions towards any service on your behalf`}
+      />
+    )
+  }
+
+  if (permissions.scopes.some((s) => RpcScope.fromString(s) != null)) {
+    return (
+      <DescriptionCard
+        role="listitem"
+        image={<AuthenticateIcon className="size-6" />}
+        title={t`Authenticate`}
+        description={t`Perform authenticated actions on your behalf`}
+      >
+        <Trans>
+          The ATproto network uses authenticated inter service communications to
```

Comment:
To be consistent with atproto.com docs, "atproto" or "ATProto":
```suggestion
          The ATProto network uses authenticated inter service communications to
```